subiquity/controllers/filesystem.py

```diff
@@ -64,6 +65,11 @@ def __init__(self, controller, restricted, timeout, cb):
         self.cb = cb
         self.state = ProbeState.NOT_STARTED
         self.result = None
+        if restricted:
+            self.kind = ErrorReportKind.RBP_FAIL
+        else:
+            self.kind = ErrorReportKind.BP_FAIL
```

Comment:
I see two kinds here, but below there are 3 scenarios (bp failed, bp timeout, bp data loading fail)  Should each be a separate enum?

Reply:
Hmm I tend to think not. It only really makes sense to have a different kind if we want to display them differently in the UI and I don't think that we really will. Can always change this later when I add the UI if my thinking changes here.